aces-nifi-processors/src/main/java/net/acesinc/nifi/processors/sockets/SocketIOProcessor.java

```diff
@@ -194,15 +195,25 @@ public void setupSockets(final ProcessContext context){
         //Try to create the socket is the URI is valid
         try {
             URI uri = new URI(url);
-            socket.set(createSocket(uri, opts));
 
             //If the scheme is HTTPS then create an SSLContext.
             if (uri.getScheme().equalsIgnoreCase(SCHEME_TYPE_HTTPS)) {
                 SSLContextService sslContextService = context.getProperty(SSL_CONTEXT_SERVICE).asControllerService(SSLContextService.class);
-                sslCon = sslContextService.createSSLContext(SSLContextService.ClientAuth.NONE);
-                opts.sslContext = sslCon;
+                sslCon = sslContextService.createContext();
```

Comment:
Ok, noted. I was anticipating a change here the the API somehow compared to before.